test/integration/suites/upgrade/README.md

```diff
@@ -25,7 +25,22 @@ To check for rotation, the SVID is written to disk at each step. It is then
 checked against the SVID for the previous step to make sure it has been
 rotated.
 
+### Maintenance
+
+#### After a SPIRE release
+
+The versions.txt should be updated to add the new version.
+
+#### Before a "Major" SPIRE Release
+
+When preparing to release a new "major" release (_minor_ release pre-1.0,
+_major_ release at and after 1.0), the versions.txt file should be updated to
```

Comment:
Suggest to remove `, _major_ release at and after 1.0` since we haven't yet discussed what supported upgrade paths will look like at that time

Reply:
:+1: